runtime/spacetime_nat.c

```diff
@@ -211,7 +209,8 @@ static void maybe_reopen_snapshot_channel(void)
   }
 }
 
-extern void caml_spacetime_automatic_save(void);
+/* Defined later */
+static void caml_spacetime_automatic_save(void);
```

Comment:
This is just a forward definition, so `static` is more appropriate (there are other like it - there are other candidates for this in `spacetime_nat.c` too, but this is the only one which ends up with an `extern` on it...)